src/Textures.cpp

```diff
@@ -1015,7 +1015,7 @@ void TextureCache::_loadBackground(CachedTexture *pTexture)
 		params.width = pTexture->realWidth;
 		params.height = pTexture->realHeight;
 		params.format = colorFormat::RGBA;
-		params.internalFormat = glInternalFormat;
+		params.internalFormat = gfxContext.convertGHQTextureFormat(u32(glInternalFormat));
```

Comment:
Also, if GLES2 does not support sized internal formats, they all can be replaced by RGBA in ContextImpl::init(), as I already did for RGBA8. That will guarantee that GLES2 will not met unsupported types in other places of code.